packages/react-avatar/SPEC.md

```diff
@@ -0,0 +1,305 @@
+
+The v8 `Persona` appears to not support a custom icon, and can only show the default person icon, or a "?" icon
+
+#### Status/Presence Badge
+
+Both components support showing a badge to indicate status, but they have different APIs. The v8 `Persona` has an enum of preset options for the presence badge. The v0 `Avatar` supports rendering a `Status` component, which is more customizable and can render an icon, and specify different sizes and colors.
+
+## Sample Code
+
+Basic examples:
+
+```jsx
+<Avatar name="Miguel Garcia" />
+<Avatar size={72} name="Mona Kane" image="./MonaKane.jpg" />
+<Avatar square icon={<IDBadgeIcon />} />
+```
+
+Displaying a badge (\*\*subject to change pending final spec for the `Badge` component):
+
+```jsx
+<Avatar name="Allan Munger" badge={<PresenceBadge status="busy" />} />
+```
+
+With active state indication:
+
+```jsx
+<Avatar name="Daisy Phillips" active="active" activeDisplay="ring-shadow" />
+<Avatar name="Robin Counts" active="inactive" />
+```
+
+## Variants
+
+### Color
+
+The Avatar supports color variants when displaying initials or an icon:
+
+- **Neutral** - Gray (default)
+- **Brand** - Brand colors from the theme
+- **Colorful** - Pick from a list of pre-defined Avatar colors. The color will be assigned based on a hash of the name
+  (to "randomly" assign a person a color). The color index can also be specified explicitly in case the use case
+  requires a different algorithm to pick the color.
+
+### Shape
+
+The Avatar natively supports a circular and square shape. However, some use cases require a hexagon shape (e.g. to represent a bot). The hexagon will not be supported directly, but it is possible to style the Avatar with a hexagon background by using `tokens`:
+
+```jsx
+<Avatar
+  icon={<ChatBotIcon />}
+  display="icon"
+  tokens={{
+    width: 'calc(var(--avatar-height) * 1.125)',
+    background: `url('${hexagonSvg}') 0px/contain no-repeat`,
+    borderRadius: '0',
+  }}
+/>
+```
+
+Where hexagonSvg is defined as:
+
+```js
+hexagonSvg =
+  'data:image/svg+xml;utf8,' +
+  '<svg width="36" height="32" viewBox="0 0 36 32" fill="none" xmlns="http://www.w3.org/2000/svg">' +
+  '<path fill="rgb(232,232,232)" d="M0.407926 17.528C-0.135976 16.5859 -0.135975 15.4141 0.407926 14.472' +
+  'L7.91541 1.46793C8.44076 0.557947 9.39444 0 10.4245 0H25.5755C26.6056 0 27.5592 0.557951 28.0846 1.46793' +
+  'L35.5921 14.472C36.136 15.4141 36.136 16.5859 35.5921 17.528L28.0846 30.5321' +
+  'C27.5592 31.4421 26.6056 32 25.5755 32H10.4245C9.39443 32 8.44076 31.4421 7.91541 30.5321L0.407926 17.528Z"/>' +
+  '</svg>';
+```
+
+## API
+
+From [Avatar.types.tsx](https://github.com/microsoft/fluentui/blob/master/packages/react-avatar/src/components/Avatar/Avatar.types.tsx):
+
+### Slots
+
+- `as` - The root element of the Avatar.
```

Comment:
nit: I don't think that we can call `as` as a slot, `as` is just a prop and behaves differently.

```suggestion
- `root` - The root element of the Avatar.
```

I would use just `root` in this case.